app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,162 @@
+/*
+ SPDX-License-Identifier: Apache-2.0
+ SPDX-FileCopyrightText: 2022-present Intel Corporation
+ */
+package org.omecproject.up4.cli;
+
+import com.google.common.collect.Maps;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.omecproject.up4.impl.Up4AdminService;
+import org.onlab.packet.Ip4Address;
+import org.onosproject.cli.AbstractShellCommand;
+import org.onosproject.net.behaviour.upf.UpfCounter;
+
+import java.util.Map;
+
+/**
+ * UP4 UE flow stats.
+ */
+@Service
+@Command(scope = "up4", name = "flow-stats",
+        description = "Read all UE flows stats")
+public class FlowsStatsCommand extends AbstractShellCommand {
+
+    private static final String SEPARATOR = "-".repeat(40);
+    private static final int DEFAULT_SLEEP_TIME = 2000; // 2 seconds
+
+    @Argument(index = 0, name = "sleep-time",
+            description = "Time between counter reads",
+            required = false)
+    int sleepTimeMs = DEFAULT_SLEEP_TIME;
+
+    @Argument(index = 1, name = "ue-session-id",
```

Comment:
Same here, maybe `-s`? The long form can be `session` for conciseness.
```suggestion
    @Argument(index = 1, name = "session",
```